ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -570,6 +571,12 @@ dhcp_relay:
   tunnel_requests_disabled: < true | false >
 ```
 
+### DOT1X
```

Comment:
`802.1X`. But please remove this from the readme, until we have something to add.

Reply:
completed.